### PR TITLE
Refactor loading fixtures by using symfony finder

### PR DESCRIPTION
**Pull request by Arminek**
I think decouple loading methods will be better approach.

### CODE REVIEW

File: src/ApiTestCase.php [hunk 1]
```diff
@@ -258,6 +258,34 @@ protected function loadFixturesFrom($source = '/')
...
+    /**
+     * @param string $source
+     *
+     * @return string
+     */
+    private function getFixtureRealPath($source)
+    {
+        $baseDirectory = $this->getFixturesFolder();
+
+        return $source = $baseDirectory.\DIRECTORY_SEPARATOR.$source;
```
Comment (pamil) [inline src/ApiTestCase.php]: `return $source =`? :)

File: test/src/Tests/Controller/SampleControllerXmlTest.php [hunk 2]
```diff
@@ -61,11 +61,12 @@ public function testGetResponseFromMockedService()
 
     public function testProductIndexResponse()
     {
-        $this->loadFixturesFrom('product.yml');
+        $this->loadFixturesFromSingleFile('product.yml');
```
Comment (lchrusciel) [inline test/src/Tests/Controller/SampleControllerXmlTest.php]: I would suggest to load here files from default directory. We have only one file there, so the result will be same, but we would check if it works properly.

Comment (pamil) [inline test/src/Tests/Controller/SampleControllerXmlTest.php]: `FromSingleFile` -> `FromFile`?

Comment (lchrusciel) [inline test/src/Tests/Controller/SampleControllerXmlTest.php]: This change make sense actually. @Arminek, can you update it?

Comment (pjedrzejewski) [inline test/src/Tests/Controller/SampleControllerXmlTest.php]: :+1:

**Pull request merged**